src/components/tab/TabGroup.js

```diff
@@ -1,18 +1,95 @@
-import React, { PureComponent } from 'react';
+import React, { createRef, PureComponent } from 'react';
+import ReactResizeDetector from 'react-resize-detector';
+import smoothScroll from 'smoothscroll-polyfill';
 import PropTypes from 'prop-types';
 import Box from '../box';
+import IconButton from '../iconButton';
+import { IconChevronLeftSmallOutline, IconChevronRightSmallOutline } from '@teamleader/ui-icons';
 import cx from 'classnames';
 import theme from './theme.css';
 
+const SCROLL_DISTANCE = 200;
+
 class TabGroup extends PureComponent {
+  scrollContainerRef = createRef();
+
+  state = {
+    canScrollLeft: false,
+    canScrollRight: false,
+  };
+
+  componentDidMount() {
+    smoothScroll.polyfill();
+    this.scrollContainerRef.current.boxNode.current.addEventListener('scroll', this.handleScroll);
+  }
+
+  componentWillUnmount() {
+    this.scrollContainerRef.current.boxNode.current.removeEventListener('scroll', this.handleScroll);
+  }
+
+  checkForScrollPosition = () => {
+    const { scrollLeft, scrollWidth, clientWidth } = this.scrollContainerRef.current.boxNode.current;
+
+    this.setState({
+      canScrollLeft: scrollLeft > 0,
+      canScrollRight: scrollLeft !== scrollWidth - clientWidth,
```

Comment:
I think you should use a `<` instead,  I think you can scroll further than the width on mac. I'll share a video with you on slack

Reply:
Will try to fix in my follow-up PR. Thanks for the manual check!